python/spec/dependabot/python/update_checker/requirements_updater_spec.rb

```diff
@@ -134,7 +134,7 @@
                 let(:requirement_txt_req_string) { "<=1.9.2,>=1.9" }
                 let(:latest_resolvable_version) { "1.10" }
 
-                its([:requirement]) { is_expected.to eq(">=1.9,<1.11.0") }
+                its([:requirement]) { is_expected.to eq(">=1.9,<=1.10") }
```

Comment:
Given that the user was requesting `1.9.2`, we're losing specificity here since we're now only specifying `1.10`, not `1.10.0`... 

I think that's okay in this case since the upstream version is `1.10`? 🤷‍♂️

Reply:
Yeah, on second thought this is probably fine--we're losing specificity in our explicit pin because the upstream package chose not to preserve the specificity of their previous releases. 

The alternative of trying to guess when to increase the specificity of upstream versioning seems like it has some edge cases where some percentage of users will always say we picked wrong.

Reply:
Yeah, that was my thinking too. I think it should be fine as is but as you said we could further iterate based on feedback 👍.